File: data/java/FIND_N_TH_ELEMENT_FROM_STERNS_DIATOMIC_SERIES_MAIN.java
public static void main(String args[]) throws IOException {
    int n_success = 0;
    List<Integer> param0 = new ArrayList<>();
    param0.add(37);
    param0.add(24);
    param0.add(13);
    param0.add(56);
    param0.add(26);
    param0.add(67);
    param0.add(82);
    param0.add(60);
    param0.add(64);
    param0.add(65);
    StringBuilder builder = new StringBuilder();
    builder.append("class,mutant,test_index,result");
    FileWriter writer = new FileWriter(args[0]);
    for(int i = 0; i < param0.size(); ++i)
    {
        try {
	        if(f_filled(param0.get(i)) == f_gold(param0.get(i)))
	        {
                builder.append("FIND_N_TH_ELEMENT_FROM_STERNS_DIATOMIC_SERIES," + args[1] + "," + i + ",SUCCESS\n");
            } else {
                builder.append("FIND_N_TH_ELEMENT_FROM_STERNS_DIATOMIC_SERIES," + args[1] + "," + i + ",FAILURE\n");
            }
        } catch (Exception e) {
            builder.append("FIND_N_TH_ELEMENT_FROM_STERNS_DIATOMIC_SERIES," + args[1] + "," + i + ",EXCEPTION\n");
        }
    }
    writer.write(builder.toString());
    writer.close();
}

File: data/python/FIND_N_TH_ELEMENT_FROM_STERNS_DIATOMIC_SERIES_MAIN.py
if __name__ == '__main__':
    param = [
    (37,),
    (24,),
    (13,),
    (56,),
    (26,),
    (67,),
    (82,),
    (60,),
    (64,),
    (65,)
        ]
    n_success = 0
    print("class,mutant,test_index,result\n")
    for i, parameters_set in enumerate(param):
        try:
            print("FIND_N_TH_ELEMENT_FROM_STERNS_DIATOMIC_SERIES," + sys.argv[1] + "," + str(i) + "," + str(f_gold(*parameters_set)))
        except:
            print("FIND_N_TH_ELEMENT_FROM_STERNS_DIATOMIC_SERIES," + sys.argv[1] + "," + str(i) + ",EXCEPTION")
